.appveyor.yml

```diff
@@ -0,0 +1,46 @@
+# To activate, change the Appveyor settings to use `.appveyor.yml`.
+install:
+  - python -m pip install --upgrade virtualenv pip setuptools tox
+
+  # Fetch the three main PyPy releases
+  - ps: (New-Object Net.WebClient).DownloadFile('https://bitbucket.org/pypy/pypy/downloads/pypy-2.6.1-win32.zip', "$env:appveyor_build_folder\pypy-2.6.1-win32.zip")
+  - ps: 7z x pypy-2.6.1-win32.zip | Out-Null
```

Comment:
2.6 is ... well very old. Why are we supporting that on Windows? That seems wasteful to say the least.